keystone_user.h

```diff
@@ -32,51 +32,51 @@
 
 
 #if __riscv_xlen == 64
-typedef __u64 u_ptr;
+typedef __u64 u_ptr_t;
 #elif __riscv_xlen == 32
-typedef __u32 u_ptr;
+typedef __u32 u_ptr_t;
 #endif 
 
 struct runtime_params_t {
-  u_ptr runtime_entry;
-  u_ptr user_entry;
-  u_ptr untrusted_ptr;
-  u_ptr  untrusted_size;
+  u_ptr_t runtime_entry;
```

Comment:
I believe we can use uintptr_t for all of these.

Reply:
Revised.